R/mcmc-diagnostics.R

```diff
@@ -580,8 +580,7 @@ acf_data <- function(x, lags) {
     abort(paste0("Too few iterations for lags=", lags, "."))
   }
 
-  data <- reshape2::melt(x, value.name = "Value")
-  data$Chain <- factor(data$Chain)
```

Comment:
Is there a reason to also get rid of making `Chain` a factor? To be honest I'm not sure why that was there in the first place (it may never have been necessary, maybe I thought `tapply` required factors), but curious if you removed that intentionally?

Reply:
I removed it intentionally. I didn’t think it was necessary as `tapply` calls `as.factor` on `INDEX` anyway.

Reply:
Ok that makes sense, thanks!